noria-server/src/controller/sql/mir/join.rs

```diff
@@ -83,7 +92,7 @@ fn pick_join_chains(
     dst: &String,
```

Comment:
Not directly related to the changes in this PR, but this can be `&str`, and changing it will avoid the `&"a".to_string()` in your test code.